File: Modul4/Soal 2/PRAK402-2310817210020-MuhammadHasbiyanRusyadi.py
n = int(input())
if n % 2 == 0:
    for angka in range(1, n+1, 2):
        print(angka, end=" ")
    print()
    for angka in range(n, 1, -2):
        print(angka, end=' ')
else:
    for angka in range(1, n+1, 2):
        print(angka, end=" ")
    print()  # Baris baru
    for angka in range(n-1, 1, -2):
        print(angka, end=" ")

File: Modul4/Soal 2/PRAK402-2310817210020-MuhammadHasbiyanRusyadi.c
#include<stdio.h>
int main (){
    int n;
    scanf("%d", &n);
    if ( n % 2 == 0){
    for(int angka = 1; angka <= n; angka += 2){
            printf("%d ", angka);} 
            printf("\n");
    for(int angka = n; angka >= 2; angka -= 2){
            printf("%d ", angka);}}
    if ( n % 2 != 0 ){
    for (int angka = 1; angka <= n; angka += 2) {
        printf("%d ", angka);} 
            printf("\n");
    for (int angka = n - 1; angka >= 2; angka -= 2) {
        printf("%d ", angka);}} }